app/src/test/kotlin/com/mapzen/erasermap/presenter/MainPresenterTest.kt

```diff
@@ -212,10 +212,10 @@ class MainPresenterTest {
         assertThat(mainController.isRoutePreviewVisible).isTrue()
     }
 
-    @Test fun onRoutePreviewEvent_shouldDisableReverseGeocode() {
```

Comment:
I think we do still want to disable reverse geocode in route preview mode. Otherwise you can wind up with a situation with two destination pins like this which can be confusing.

![image](https://cloud.githubusercontent.com/assets/464123/14327988/c5cc13ba-fc02-11e5-9759-4d0e607d5e54.png)